src/chariot/database/SQLiteDatabase.py

```diff
@@ -0,0 +1,73 @@
+import sqlite3
+import csv
+import os
+from typing import List
+
+from chariot.database.Database import Database
+
+
+class SQLiteDatabase(Database):
+    def __init__(self, db_path: str = '../data/iot_database.db'):
+        Database.__init__(self, db_path)
+        self.connectToDB()
+
+    def initializeDB(self):
+        cursor = self.conn.cursor()
+        cursor.execute(
+            'CREATE TABLE IF NOT EXISTS IOTD('
+            'id INTEGER PRIMARY KEY, ' +
+            'db_insertion_time DATETIME DEFAULT CURRENT_TIMESTAMP, ' +
+            'relative_time BIGINT, ' +
+            'freeform TEXT)')
+        self.conn.commit()
+
+    def connectToDB(self):
+        # Make the data directory if it does not exist
+        dirname = os.path.dirname(self.db_path)
```

Comment:
This cannot be the flow. We can't store the user's database locally on our server. The user should provide a remote link to the database through which we connect.